src/libs/actions/PersonalDetails.js

```diff
@@ -129,7 +130,7 @@ function fetchPersonalDetails() {
             // Set my personal details so they can be easily accessed and subscribed to on their own key
             Onyx.merge(ONYXKEYS.MY_PERSONAL_DETAILS, myPersonalDetails);
         })
-        .catch(error => console.debug('Error fetching personal details', error));
+        .catch(error => Log.hmmm('Error fetching personal details', error));
```

Comment:
If we are going to catch and log API promise rejections then perhaps we should do it in a global way so we don't have to apply these inconsistently? Same comment about local currency, reportStuff, etc calls below.

Reply:
Okay, added a global server log for API errors [here](https://github.com/Expensify/App/blob/a51f1d4a86295c2c3d97982f7c894b8fdb9dd26c/src/libs/API.js#L178)